### DISCUSSION TITLE
Are we able to prevent hotlinking of files?

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17709] [Archive]
I need to implement a hotlinking prevention for uploaded files in Directus.

Is this possible at the moment?
If not: how or where can I hook into directus, to write the script my own?

**Comment by Zack-Heisnberg** [comment]
For a better practice , and performance you should serve the app through Nginx and prevent hotlinking with a config like that

```
location ~ .(gif|png|jpe?g)$ {
valid_referers none blocked yourdomain.com *.yourdomain.com;
if ($invalid_referer) {return 403;}}
```
